### DISCUSSION TITLE
drg in barotrauma server crash when trying to revive dwarf clown passanger on flooded humpback

### DISCUSSION THREAD

**Discussion by DrBruhman** [FakeFishGames/Barotrauma#13752] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

title

### Reproduction steps

mods: human dwarfs, easy pressure, flooded humpback(submarine), i need more boolets

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.3.0.2

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
p2p error smt
```

**Comment by Regalis11** [comment]
Thank you for the report! I'm afraid there's not much we can do without the error message or more information on how exactly the issue can be reproduced. And if it's an issue caused by a mod, it may be that it's something that needs to be addressed by the creator of the mod, not something we can fix.

**Reply by DrBruhman** [reply]
my server crashed two times the first was my friend joining server and subscribing to mods the second is described in ticket. 
idk what is wrong in mods they changing only human prefab, organ damage and weapons craft recipes

**Reply by Regalis11** [reply]
Unfortunately there is not be enough information here for us to address the issue. I think we need either the error message and/or exact repro step to diagnose it.

**Reply by DrBruhman** [reply]
now server crashes without error, even without mods. also something is broken with other servers that i join get shuted after some time. and 1 server name tells that barotrauma servers get ddosed.
